src/SchemaGenerator.cpp

```diff
@@ -173,6 +179,30 @@ static_assert(graphql::internal::MinorVersion == )cpp"
 };
 
 )cpp";
+			firstValue = true;
+
+			headerFile << R"cpp(constexpr std::array<std::string_view, )cpp"
```

Comment:
It looks like we don't necessarily include `<array>` or `<string_view>` with this header. The PR build failed on Windows saying that those types were undefined. If we want to move this to the header, I think you'll also need to add those `#include` directives to the `headerFile` preamble.

Reply:
Will fix up. Apologies - I only have a Linux set of machines to run the tests on here.